src/runtime/recipe/recipe-resolver.ts

```diff
@@ -20,25 +20,33 @@ import {Recipe, IsValidOptions} from './recipe.js';
 import {ConnectionConstraint, InstanceEndPoint} from './connection-constraint.js';
 import {SlotConnection} from './slot-connection.js';
 import {SlotUtils} from './slot-utils.js';
+import {Continuation} from './walker.js';
 
 export class ResolveWalker extends RecipeWalker {
+  options: IsValidOptions;
```

Comment:
can we make it private?

Reply:
Done.

I'd prefer to pass the options around explicitly, but the RecipeWalker & Walker classes don't currently support that. Should I keep track of this or file a bug?